static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -245,3 +245,321 @@
         }
     }
 }
+
+
+// New gallery stuff
+
+// TODO: put in pasteup
+$multimedia-support-5: #161616;
+
+// The main image needs to take up 100vh and the slim header needs to sit just on top of it.
+.l-header--new-gallery {
+    position: absolute;
+    z-index: 1;
+    right: 0;
+    left: 0;
+}
+
+
+// TODO: When the time comes, remove `.new-gallery` class completely and remove old gallery styling
+.new-gallery {
+    .ad-slot--dark {
+        background-color: $multimedia-support-5;
+    }
+
+    .content__main-column {
+        max-width: none;
+
+        @include mq(tablet) {
+            margin-left: gs-span(3);
+        }
+    }
+
+    @include mq(tablet) {
+        .content__dateline,
+        .content__labels,
+        .byline,
+        .content__meta-container,
+        .meta__break,
+        .content__headline,
+        .content__standfirst {
+            margin-left: gs-span(3);
+            margin-right: gs-span(1);
+            max-width: gs-span(8);
+        }
+        .byline, .content__dateline {
+            border: 0;
+        }
+        .content__standfirst {
+            padding-bottom: 0;
+            padding-top: 0;
+            &:before {
+                content: none;
+            }
+        }
+    }
+
+
```

Comment:
Double return inconsistent with single return below. I think we tend to have single lines after blocks. But should at least be consistent in same file.